WordleGame/WordleGame/SettingsViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SettingsViewController.swift
+//  WordleGame
+//
+//  Created by Alex on 15.05.22.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    @IBOutlet weak var soundSwitch: UISwitch!
+    @IBOutlet weak var vibrationSwitch: UISwitch!
+    
+    @IBOutlet weak var languagePicker: UIPickerView!
+    
+    let data = ["US", "RUS"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        languagePicker.dataSource = self
+        languagePicker.delegate = self
+        
+        soundSwitch.isOn = UserDefaults.standard.bool(forKey: "Sound Setting")
+        vibrationSwitch.isOn = UserDefaults.standard.bool(forKey: "Vibration Setting")
```

Comment:
Why you haven't created these values in `UserDefaultsKey`?

Reply:
Because I thought it was a simple UserDefault type and better shown in code not in service.

Reply:
It's still better to add these values to `UserDefaultsKey` and move them to service so it won't be a problem if you misspell the key